docs/topics/cache.txt

```diff
@@ -435,8 +452,39 @@ of 60 seconds, and a maximum capacity of 1000 items::
         }
     }
 
-Invalid arguments are silently ignored, as are invalid values of known
-arguments.
+Here, a ``python-memcached`` based backend is being configured with an object
```

Comment:
Here's an example configuration for a ``python-memcached`` based backend with an object...